File: project/module-script/src/test/resources/samples/fluxon/variable-expression.fs
total = 0
for i in 0..<__ITER__ {
 numerator = ((&base + &i) * (&multiplier - (&i % &modulus))) + &offset
 total += ((&numerator - (&numerator % &divisor)) / &divisor) + (&bias * (&i % 3))
}
&total

File: project/module-script/src/test/resources/samples/javascript/variable-expression.js
var total = 0;
for (var i = 0; i < __ITER__; i++) {
    var numerator = ((base + i) * (multiplier - (i % modulus))) + offset;
    total += ((numerator - (numerator % divisor)) / divisor) + (bias * (i % 3));
}
total;
